ipmininet/ipnet.py

```diff
@@ -549,6 +550,64 @@ def ping6Pair(self):
            return: ploss packet loss percentage"""
         return self.pingPair(use_v4=False)
 
+   
+    def runFailurePlan(self,failure_plan):
+        """this function run a failure plan
+            the plan have to be a list of tuple with the name of routeur between
+            which the link have to be down
+            example of plan:
+            [("R1","R2"),("R8","R9"),("R3","R1")]
+            This will return a list of IPIntf 
```

Comment:
Please use "return: " and specify that the interfaces that are returned are the stopped interfaces